src/main/java/in/muthukumari/validator/BankDetailValidator.java

```diff
@@ -111,20 +117,25 @@ public static boolean isValidMobileNumber(CustomerBankDetail customer) {
 	 * 
 	 * @param customer
 	 * @return
-	 * @throws ClassNotFoundException
-	 * @throws SQLException
+	 * @throws InvalidException
 	 */
-	public static boolean isValidIfscCode(CustomerBankDetail customer) throws ClassNotFoundException, SQLException {
+	public static boolean isValidIfscCode(CustomerBankDetail customer) throws InvalidException {
 		boolean isValidIfsc = false;
-		Map<String, String> branchAndIfscList = BankDAO.getBranchNameAndIfscList(customer.getBankName());
-		if (branchAndIfscList.containsKey(customer.getBranchName())) {
-			String ifsc = branchAndIfscList.get(customer.getBranchName());
-			if (ifsc.equals(customer.getIfscCode())) {
-				isValidIfsc = true;
-			} else {
-				isValidIfsc = false;
+		Map<String, String> branchAndIfscList = null;
+		try {
+			branchAndIfscList = BankDAO.getBranchNameAndIfscList(customer.getBankName());
+			if (branchAndIfscList.containsKey(customer.getBranchName())) {
+				String ifsc = branchAndIfscList.get(customer.getBranchName());
+				if (ifsc.equals(customer.getIfscCode())) {
+					isValidIfsc = true;
+				} else {
+					isValidIfsc = false;
+				}
 			}
+		} catch (DBException e) {
```

Comment:
validator only throw validator exception not others

Reply:
I have changed